### PR TITLE
Tune up BasicViews

### PR DESCRIPTION
**Pull request by rynowak**
- Prune files
- Add error logging to console
- Do a post with form data instead of query string
- Require antiforgery token
- Update client to send antiforgery token

### CODE REVIEW

**Comment by rynowak** [comment]
FYI only - will merge immediately.

/cc @SajayAntony - this technique with powershell is actually pretty easy to do :+1:

File: testapp/BasicViews/Views/Home/HtmlHelpers.cshtml [hunk 1]
```diff
@@ -18,4 +18,6 @@
         @Html.EditorFor(p => p.BirthDate)
     </div>
     <input type="submit" />
+
+    @Html.AntiForgeryToken()
```
Comment (rynowak) [inline testapp/BasicViews/Views/Home/HtmlHelpers.cshtml]: This isn't an edit, I deleted `SuppressAntiforgery.cshtml` and moved `Index.cshtml` -> `HtmlHelpers.cshtml`

File: testapp/BasicViews/loadtest.ps1 [hunk 2]
```diff
@@ -1,6 +1,44 @@
-param([int] $iterations = 3000)
+param(
```
Comment (rynowak) [inline testapp/BasicViews/loadtest.ps1]: This still runs the same way - `.\loadtest.ps1`

File: testapp/BasicViews/loadtest.ps1 [hunk 3]
```diff
@@ -1,6 +1,44 @@
...
+}
+
+Write-Host -ForegroundColor Green Got token
+
+$set_cookie = $response.Headers["Set-Cookie"]
+$cookie_name = $set_cookie.Substring(0, $set_cookie.IndexOf('='))
+$cookie_value = $set_cookie.Substring($cookie_name.Length + 1, $set_cookie.IndexOf(';') - $cookie_name.Length - 1)
+
+$form_name = "__RequestVerificationToken"
+$form_value = $response.Forms[0].Fields[$form_name]
```
Comment (rynowak) [inline testapp/BasicViews/loadtest.ps1]: So nice that powershell's OM provides this.

**Comment by SajayAntony** [comment]
NICE :+1:

**Comment by SajayAntony** [comment]
Curious, why are there unmerged changes?

**Comment by rynowak** [comment]
> Curious, why are there unmerged changes?

We don't use the big green button. If you have to resolve conflicts when rebasing it will appear that way in github.

**Comment by SajayAntony** [comment]
:scream: - 
sounds good

**Pull request closed without merging**